frontend/pages/beta/model/[modelId]/release/[semver].tsx

```diff
@@ -34,34 +34,35 @@ export default function Release() {
 
   if (error) return error
 
+  if (!release || (isReleaseLoading && isActiveReviewsLoading)) {
```

Comment:
Should we be handling `!release` differently?

Reply:
If we don't have a release and it hasn't errored then it just means it's loading, so makes sense to put in this block imo. Also means we don't have conditional logic down below in our main JSX block.